src/models/AccordionItem.php

```diff
@@ -7,28 +7,61 @@
  *
  * @property string $Title
  * @property string $Content
+ * @property int $AccordionID
  * @property int $SortOrder
  * @property int $PageID
  * @method Page Page()
  */
 class AccordionItem extends DataObject
 {
     private static $db = [
-        'Title'     => 'Varchar(255)',
-        'Content'   => 'HTMLText',
-        'SortOrder' => 'Int',
+        'Title'         => 'Varchar(255)',
+        'Content'       => 'HTMLText',
+        'AccordionID'   => 'Int(1)',
```

Comment:
Please don't use `ID` at the end, it confuses with a `has_one` relation. I'd suggest go with `Set`?